sklearn/cluster/k_means_.py

```diff
@@ -868,14 +823,14 @@ class KMeans(BaseEstimator, ClusterMixin, TransformerMixin):
     >>> import numpy as np
     >>> X = np.array([[1, 2], [1, 4], [1, 0],
     ...               [10, 2], [10, 4], [10, 0]])
-    >>> kmeans = KMeans(n_clusters=2, random_state=0).fit(X)
+    >>> kmeans = KMeans(n_clusters=2, random_state=1234).fit(X)
```

Comment:
Why changing this?

Reply:
idk :/ 
It was a while ago... I'll check why I did that.